README.md

```diff
@@ -86,15 +71,12 @@ This can be either a badge, a virtual item from the game's economy (sword, coins
 ## Getting Started
 > NOTE: LevelUp depends on SOOMLA's other modules: Core, Store, and Profile. This document assumes that you are new to SOOMLA and have not worked with any of the other SOOMLA modules. If this is not the case, and you already *have* some or all of the other modules, please follow these directions only for the modules you are missing and of course, for the **LevelUp** module.
 
-1. Download the following unitypackages:
-    - [soomla-unity3d-core](http://library.soom.la/fetch/unity3d-core/latest?cf=github)  
-    - [unity3d-store](http://library.soom.la/fetch/unity3d-store/latest?cf=github)  
-    - [unity3d-profile](http://library.soom.la/fetch/unity3d-profile/latest?cf=github)  
-    - [unity3d-levelup](http://library.soom.la/fetch/unity3d-levelup/latest?cf=github)  
-2. Double-click on them following the order they appear above. It'll import all the necessary files into your project.
-> If you are completely new to SOOMLA you can use the [All-in-One](https://github.com/soomla/unity3d-levelup/raw/master/soomla-unity3d-all-in-one.unitypackage) unitypackage which contains all the above packages in one package. Importing it is equivalent to preforming the above steps.
+1. Clone this repository recursively: `git clone --recursive https://github.com/soomla/unity3d-levelup.git`
+
+2. Run `./build_all` from project directory
+
+3. Import created binaries from `build` directory.
```

Comment:
@Blinkq there is no build directory here. Please make sure the instructions you write actually work.